src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Customizations/Indexes/FieldBuilder.cs

```diff
@@ -53,17 +54,22 @@ public static IList<Field> BuildForType<T>(IContractResolver contractResolver)
             var fields = new List<Field>();
             foreach (JsonProperty prop in contract.Properties)
             {
+                IList<Attribute> attributes = prop.AttributeProvider.GetAttributes(true);
+                if (attributes.Any(attr => attr is JsonIgnoreAttribute))
+                {
+                    continue;
+                }
+
```

Comment:
should we add a regression test for this?

Reply:
@Yahnoosh There are already test changes for this. Look at the test model class.

Reply:
thanks

---
In reply to: [166069859](https://github.com/Azure/azure-sdk-for-net/pull/4036#discussion_r166069859) [](ancestors = 166069859)